utilities.js

```diff
@@ -0,0 +1,317 @@
+// *** FOR JEST TESTING ***
+
+// Helper function for matching input objects with objects returned from DB queries
+const matchesDatabase = (inputObject, databaseObject) => {
+    let areAllPropertiesMatched = true;
+
+    for (const prop in inputObject) {        
+        // convert prop to snake case to match DB column name in query return
+        let propForDatabaseObject = prop;
+
+        if (prop.match(/[A-Z]/g) !== null) {
+            const charArray = prop.split('');
+            const finalCharArray = charArray.map(char => {
+                if (char.match(/[A-Z]/g) !== null) {
+                    return "_" + char.toLowerCase();
+                }
+                return char;
+            })
+            propForDatabaseObject = finalCharArray.join('');
+        }
+
+         // Account for PT to UTC adjustment in DB
+        if (
+                databaseObject[propForDatabaseObject] instanceof Date &&
+                inputObject[prop] + 'T07:00:00.000Z' !== databaseObject[propForDatabaseObject].toISOString()
+            ) {
+                areAllPropertiesMatched = false;
+                return areAllPropertiesMatched;
+            }
+
+        if (!(databaseObject[propForDatabaseObject] instanceof Date) &&
+            inputObject[prop] !== databaseObject[propForDatabaseObject]) {
+            areAllPropertiesMatched = false;
+            return areAllPropertiesMatched;
+        }
+    }
```

Comment:
Have you looked into whether `jest` already built something like this for checking if an object matches another object?

Reply:
I would use `toStrictEqual` but the object keys are different in the database versus `utilities.js` (camelCase versus snake_case). Plus the time stamp gets converted to UTC in the database so it is a slightly different time stamp than the input date when I query for it. I could change the time zone for my DB to Pacific, but I didn't know if that was the right thing to do. I should have asked. Is that the right thing to do?

Reply:
I think you're right to keep timestamp on DB as UTC. Look into how to get the timezone to change so your tests can match.